doonut-app-external-api/src/main/kotlin/com/doonutmate/calendar/controller/CalendarController.kt

```diff
@@ -0,0 +1,39 @@
+package com.doonutmate.calendar.controller
+
+import com.doonutmate.calendar.controller.dto.CalendarResponse
+import com.doonutmate.calendar.controller.dto.CalendarResult
+import com.doonutmate.calendar.service.CalendarAppService
+import com.doonutmate.oauth.configuration.Authorization
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.time.Instant
+
+@RestController
+@Tag(name = "Calendar", description = "캘린더 API")
+@RequestMapping("/calendars")
+class CalendarController(
+    private val calendarAppService: CalendarAppService,
+) {
+    @Operation(
+        summary = "커뮤니티 캘린더 모아보기 ",
+        description = "1. 첫요청시 time은 null \n" +
+            "2. 이후 요청시 마지막 속성의 updatedAt을 time 파라미터로 사용\n" +
+            "3. 다음 조회가 불가능한 경우 hasNext가 false\n" +
+            "4. size가 null이면 10, null이 아니면 size크기만큼 조회됨",
```

Comment:
Java 17의 Text Blocks처럼 Kotlin에도 [String Literal Multiline strings](https://kotlinlang.org/docs/strings.html#multiline-strings) 기능이 있어 여러 줄의 문자열을 손쉽게 표현할 수 있습니다!  

마크다운에서와 같이 줄 끝에 띄어쓰기 2번을 같이 입력하면 줄바꿈이 되더라구요.

```suggestion
                description = """ 
            1. 첫요청시 time은 null  
            2. 이후 요청시 마지막 속성의 updatedAt을 time 파라미터로 사용  
            3. 다음 조회가 불가능한 경우 hasNext가 false  
            4. size가 null이면 10, null이 아니면 size크기만큼 조회됨  
            """
```

스웨거 상에선 아래와 같이 표현됩니다!  

![스크린샷 2024-05-18 오전 9 55 30](https://github.com/doonutmate/doonut-server/assets/52141636/102fe179-19a5-4d2f-a39b-bec2ceaa1fbd)

Reply:
이런게 있었군요...! 방금 스웨거로 확인하니까 이쪽이 가독성이 훨씬 괜찮은거같네요.